specification.json

```diff
@@ -45,7 +45,7 @@
         {
             "id": "Requirement 1.1.5",
             "machine_id": "requirement_1_1_5",
-            "content": "The API MUST provide a function for retrieving the metadata field of the configured `provider`.",
+            "content": "The `API` MUST provide a function for retrieving the metadata field of the configured `provider`.",
```

Comment:
If we are going to monospace this it should be consistent across all entries. It's probably not necessary to make that change though

Reply:
I agree. I noticed a few small inconsistencies like that. I'll create a separate PR that addresses some of the style inconsistencies, and maybe even see if we can do some basic custom linting.